servo/connectors/kubernetes.py

```diff
@@ -2003,6 +2016,578 @@ async def get_restart_count(self) -> int:
+    image: str
+    image_pull_policy: Optional[str]
+    lifecycle: Any
+    liveness_probe: Any
+    name: str
+    ports: Optional[List[RolloutV1ContainerPort]]
+    readiness_probe: Any
+    resources: Optional[RolloutV1ResourceRequirements]
+    security_context: Any
+    startup_probe: Any
+    stdin: Optional[bool]
+    stdin_once: Optional[bool]
+    termination_message_path: Optional[str]
+    termination_message_policy: Optional[str]
+    tty: Optional[bool]
+    volume_devices: Any
+    volume_mounts: Any
+    working_dir: Optional[str]
+
+class RolloutV1PodSpec(RolloutBaseModel):
+    active_deadline_seconds: Optional[int]
+    affinity: Any
+    automount_service_account_token: Optional[bool]
+    containers: List[RolloutV1Container]
+    dns_config: Any
+    dns_policy: Optional[str]
+    enable_service_links: Optional[bool]
+    ephemeral_containers: Any
+    host_aliases: Any
+    host_ipc: Optional[bool]
+    host_network: Optional[bool]
+    host_pid: Optional[bool]
+    hostname: Optional[str]
+    image_pull_secrets: Any
+    init_containers: Optional[List[RolloutV1Container]]
+    node_name: Optional[str]
+    node_selector: Optional[Dict[str, str]]
+    overhead: Optional[Dict[str, str]]
+    preemption_policy: Optional[str]
+    priority: Optional[int]
+    priority_class_name: Optional[str]
+    readiness_gates: Any
+    restart_policy: Optional[str]
+    runtime_class_name: Optional[str]
+    scheduler_name: Optional[str]
+    security_context: Any
+    service_account: Optional[str]
+    service_account_name: Optional[str]
+    share_process_namespace: Optional[bool]
+    subdomain: Optional[str]
+    termination_grace_period_seconds: Optional[int]
+    tolerations: Any
+    topology_spread_constraints: Any
+    volumes: Any
+
+class RolloutV1PodTemplateSpec(RolloutBaseModel):
+    metadata: RolloutV1ObjectMeta
+    spec: RolloutV1PodSpec
+
+class RolloutSpec(RolloutBaseModel):
+    replicas: int
+    selector: RolloutV1LabelSelector
+    template: RolloutV1PodTemplateSpec
+    min_ready_seconds: Optional[int]
+    revision_history_limit: Optional[int]
+    paused: Optional[bool]
+    progress_deadline_seconds: Optional[int]
+    restart_at: Optional[datetime.datetime]
+    strategy: Any
+
+class RolloutBlueGreenStatus(RolloutBaseModel):
+    active_selector: Optional[str]
+    post_promotion_analysis_run: Optional[str]
+    post_promotion_analysis_run_status: Any
+    pre_promotion_analysis_run: Optional[str]
+    pre_promotion_analysis_run_status: Any
+    preview_selector: Optional[str]
+    previous_active_selector: Optional[str]
+    scale_down_delay_start_time: Optional[datetime.datetime]
+    scale_up_preview_check_point: Optional[bool]
+
+class RolloutStatusCondition(RolloutBaseModel):
+    last_transition_time: datetime.datetime
+    last_update_time: datetime.datetime
+    message: str
+    reason: str
+    status: str
+    type: str
+
+class RolloutStatus(RolloutBaseModel):
+    HPA_replicas: Optional[int]
```

Comment:
Use the alias functionality to rename the field. We don't want fields with non-idiomatic names